pkg/virt-controller/services/template.go

```diff
@@ -1547,44 +1546,64 @@ func getIfaceByName(vmi *v1.VirtualMachineInstance, name string) *v1.Interface {
 	return nil
 }
 
-func getCniAnnotations(vmi *v1.VirtualMachineInstance) (cniAnnotations map[string]string, err error) {
-	ifaceListMap := make([]map[string]string, 0)
-	cniAnnotations = make(map[string]string, 0)
+func getMultusCNIAnnotation(vmi *v1.VirtualMachineInstance) (string, error) {
+	var ifaceListMap []map[string]string
```

Comment:
This name tell me little. Except that this is somehow related to interfaces, I am clueless without digging deep in what is put in it later on.
TBD

Reply:
Is it better now ?